onnxruntime/test/optimizer/cse_test.cc

```diff
@@ -96,13 +96,8 @@ TEST(CseTests, SimpleTestTraining) {
                   .IsOK());
 
   GraphTransformerManager graph_transformation_mgr(1);
-  // need to declare variables to avoid build error after making
-  // weights_to_train and updated_weight_names as non-const
-  std::unordered_set<std::string> weights_to_train;
-  std::unordered_map<std::string, std::string> updated_weight_names;
-  std::unordered_map<std::string, training::TrainingSession::PartitionInfo> weight_partition_info;
   auto transformers_to_register = training::transformer_utils::GeneratePreTrainingTransformers(
-      TransformerLevel::Level1, weights_to_train, {}, CPUExecutionProvider(CPUExecutionProviderInfo()), updated_weight_names, weight_partition_info);
+      TransformerLevel::Level1, {}, {}, CPUExecutionProvider(CPUExecutionProviderInfo()));
```

Comment:
still need to pass in the proper weigths_to_train? 
constant folding still need this.

Reply:
in this test, no valid weights are provided. The removed comments said weights_to_train was added for previously change we making GeneratePreTrainingTransformers accepting non-const reference of weights_to_train. Now I change GeneratePreTrainingTransformers  to accept const reference back.